lib/SourceMapDevToolPlugin.js

```diff
@@ -283,17 +283,22 @@ class SourceMapDevToolPlugin {
 
 								if (task) {
 									const modules = task.modules;
-
+									const chunk = fileToChunk.get(file);
+									const { name } = (chunk && chunk.getEntryOptions()) || {
+										name: "*"
+									};
+									const namespaceEnsure = namespace.replace(/\[name\]/g, name);
 									for (let idx = 0; idx < modules.length; idx++) {
 										const module = modules[idx];
+										const moduleUnique = module + "|" + name;
```

Comment:
I think a clearer name for this variable would be `moduleUniqueKey`.

Reply:
That's a great suggestion

Reply:
Done